charts/folder-config-connector/values.yaml

```diff
@@ -0,0 +1,61 @@
+# -- detailed description of the parameters: https://cloud.google.com/config-connector/docs/reference/resource-docs/resourcemanager/folder
+
+# -- Single folder name. If set, "folders" key has to be unset. 
+name: ""
+
+# -- Required if name is set. The folder's display name 
+# displayName: "example-folder"
+
+# -- Optional. Labels for folders or shared by all projects under "projects" key
+# labels:
+#   labelKey: labelValue
+
+# -- Optional. Annotations for folders or shared by all projects under "projects" key
+# annotations:
+#   annotationKey: annotationValue
+
+# -- Optional. The organization that folder belongs to.
+# organizationRef:
+# -- Optional. Allowed value: The `name` field of an `Organization` resource.
+#   external: "123456789"
+# -- Optional. Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+#   namespace: some-ns
+# -- Optional. Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+#   name: demo-org
+
+# -- Optional. The folder that this resource belongs to.
+# folderRef:
+# -- Optional. Allowed value: The `folderId` field of a `Folder` resource
+#   external: "123456789"
+# -- Optional. Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+#   namespace: some-ns
+# -- Optional. Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+#   name: demo-folder
+# -- Optional. The service-generated name of the resource. Used for acquisition only. Leave unset to create a new resource.
+# resourceID: folderID
+
+# -- List of folders to create. If set, "name" key has to be unset.
+folders: {}
```

Comment:
dwa komentarze: 

1. jezeli to jest lista to default value powinna byc []
2. Zdecydowanie powinna to być mapa, tak żeby dało się overridować. np argo nie potrafi zrobic takiego deep merga, żeby scalac listy

Reply:
list has been migrated to map